assets/style.css

```diff
@@ -0,0 +1,603 @@
+  color: #326891;
+  text-decoration: underline;
+}
+
+.main-content .author-name a {
+  text-decoration: underline;
+  text-decoration-color: rgba(180, 177, 177, 0.589);
+}
+
+.space-astronomy a:hover {
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+.most-popular a:hover {
+  color: #666;
+}
+
+.footer-bottom a:hover {
+  text-decoration: underline;
+}
+
+.editor-pick-aside a:hover {
+  text-decoration: underline;
+}
+
+.main-content p a:hover {
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.date-share-icons p {
+  font-size: 13px;
+}
+
+.date-share-icons ul {
+  flex: 70%;
+  justify-content: flex-end;
+}
+
+.social-icons {
+  display: inline-block;
+  margin: 0 9px;
+  font-size: 14px;
+  background-color: rgba(209, 208, 208, 0.623);
+  padding: 6px;
+  border-radius: 50%;
+  border: 1px solid rgba(209, 208, 208, 0.623);
+}
+
+.fa-facebook {
+  padding: 6px 9px;
+}
+
+.fa-bookmark,
+.fa-comment-alt {
+  font-size: 25px;
+  padding: 0 7px;
+  color: rgba(0, 0, 0, 0.603);
+  display: inline-block;
+  font-weight: 100;
+}
+
+.book-comment {
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 20px;
+  margin: 0 10px;
+}
+
+.date-share-icons .social-icons:hover {
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.theory-of-inflation {
+  width: 70%;
+  margin: 0 auto;
+}
+
+.theory-of-inflation h4,
+.theory-of-inflation p {
+  font-size: 16px;
+}
+
+.theory-of-inflation h4 {
+  margin-bottom: -8px;
+}
+
+.theory-of-inflation p {
+  font-size: 14px;
+  line-height: 16px;
+  color: rgb(107, 103, 103);
+}
+
+.menu li {
+  font-size: 14px;
+  padding-bottom: 10px;
+}
+
+.subscribe-section li {
+  padding-bottom: 15px;
+  padding-left: -20px;
+}
+
+.first-row ul li,
+.second-row ul li {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  justify-content: space-between;
+}
+
+.first-row ul,
+.second-row ul {
+  margin: 0;
+  padding: 0;
+}
+
+.telescope-img figcaption {
+  color: #666;
+  font-size: 17px;
+  line-height: 20px;
+  width: 97%;
+  padding-top: 8px;
+}
+
+.space-astronomy figure,
+.editor-pick figure {
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+.first-row ul li figure,
+.second-row ul li figure {
+  margin-left: 0;
+  padding-right: 10px;
+}
+
+.footer-heading {
+  justify-content: space-between;
+}
+
+.space-astronomy figure figcaption,
+.editor-pick figure figcaption {
+  width: 290px;
+}
+
+.first-row ul li figure figcaption,
+.second-row ul li figure figcaption {
+  font-size: 13px;
+  color: gray;
+  line-height: 16px;
+  font-weight: 100;
+}
+
+/* Video sec */
+.video p span {
+  font-size: 13px;
+}
+
+.first-row ul li figure figcaption span,
+.second-row ul li figure figcaption span {
+  color: #000;
+  font-weight: 500;
+}
+
+.second-row {
+  border-bottom: 1px solid #000;
+  width: 80px;
+}
+
+.theory-of-inflation .image-courtsey {
+  color: #000;
+}
+
+/* Video Section */
+
+.video p {
+  color: #666;
+  margin-top: 3px;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-xyz{
  color: #666;
}
```
**Html file**

```
<div class="color-xyz">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_